code/workspaces/web-app/src/components/common/RoutePermissionWrapper.js

```diff
@@ -13,7 +13,7 @@ class RoutePermissionWrapper extends Component {
       return () => (<CircularProgress />);
     }
 
-    if (!fetching && value.includes(this.props.permission)) {
+    if (!fetching && (!this.props.permission || value.includes(this.props.permission))) {
```

Comment:
I do take code climates point here the `if` condition could be a sensibly named function to hide the complexity at this level.
```
if (dataLoadedAndUserHasPermissionToView(fetching, value, permission)
```